scripts/Makefile.build

```diff
@@ -277,7 +277,7 @@ $(obj)/%.lst: $(src)/%.c FORCE
 # Compile Rust sources (.rs)
 # ---------------------------------------------------------------------------
 
-rust_allowed_features := core_ffi_c
+rust_allowed_features := core_ffi_c,explicit_generic_args_with_impl_trait
```

Comment:
Similarly this?

This feature was stabilized in 1.63, so this might not be here for long.

Reply:
Don't worry about it -- Lina has a patch enabling it too, so I can pick that one first: https://lore.kernel.org/rust-for-linux/20230224-rust-new_uninit-v1-1-c951443d9e26@asahilina.net/

Reply:
No worries either, it will go away in the merge cycle.